catkin_virtualenv/cmake/catkin_generate_virtualenv.cmake

```diff
@@ -17,14 +17,27 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 function(catkin_generate_virtualenv)
-  cmake_parse_arguments(ARG "PYTHON3" "" "" ${ARGN})
+  set(oneValueArgs PYTHON_VERSION_MAJOR USE_SYSTEM_PACKAGES ISOLATE_REQUIREMENTS)
+  cmake_parse_arguments(ARG "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN} )
```

Comment:
`"${multiValueArgs}"` is just being explicit/future-proofing, or are you missing some args?

Reply:
just being explicit that that's what cmake_parse_argument expects. Undefined args get subbed as "".